kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -16,16 +16,46 @@
  * code into multiple, classes.
  *
  * <p>TODO: Have a read only chunk info for read only chunks so we don't accidentally update it.
- * TODO: Add a state machine for a chunk?
+ *
+ * <p>TODO: Add a minOffset field to chunk and snapshot metadata.
```

Comment:
We should do this now at the same time we're adding the max offset, or remove the todo if it's no longer needed.

Reply:
Since we're already refactoring this class to add `maxOffset` we should add the `minOffset `at the same time instead of a todo. It's easier to add it now, and it should only be a few additional lines. Then we won't need to come back to it later or track the todo.

Reply:
Dropped the TODO for now since we may not even need this field.